backend/node/genvm/types.py

```diff
@@ -21,15 +55,19 @@ class ExecutionResultStatus(Enum):
 @dataclass
 class PendingTransaction:
     address: str  # Address of the contract to call
-    method_name: str
-    args: list
+    calldata: bytes
+
+    def to_dict(self):
+        return {
+            "address": self.address,
+            "calldata": str(base64.b64encode(self.calldata), encoding="ascii"),
```

Comment:
maybe it's a good idea for code organiztion to have a function in the GenVM module called something like "encode_calldata" so that we don't repeat always `b64encode...` and we have one source of truth to integrate with the GenVM

Reply:
Considering that next pr will completely replace genvm folder and is more than half done... I am not sure that it is required right now